.circleci/config.yml

```diff
@@ -7,62 +7,76 @@ version: 2.1
 
 commands:
   go_save_cache:
-    description: "Save $GOHOME directory into the cache for the actual revision"
+    description: Save $GOPATH directory as workspace
     steps:
-      - save_cache:
-          key: env-{{ .Revision }}
+      - persist_to_workspace:
```

Comment:
Workspace are meant to be used to share code with downstream steps and provide retry due to their persistence.

Reply:
```suggestion
      # circleci workspace are meant to be used to share data between jobs
      # https://circleci.com/docs/workspaces/
      - persist_to_workspace:
```